lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
When setting external content via `#external_content=`, would it make sense to first set the content to a 0 byte string? The purpose would be to avoid dangling references to binary data.